commit 1bcb43b0efd18f316422942eb64069ffd7e11ce0 (refs/changes/75/5375/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-01-18T11:41:14+00:00 (9 months ago)
    
    MANTA-3921 manta-init leaves determining missing images as an exercise for the reader
    Reviewed by: Rui <rui@joyent.com>
    Approved by: Rui <rui@joyent.com>

diff --git a/cmd/manta-init.js b/cmd/manta-init.js
index 87355c8..7aa6bef 100755
--- a/cmd/manta-init.js
+++ b/cmd/manta-init.js
@@ -7,7 +7,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 /*
@@ -222,8 +222,10 @@ function findLatestImage(service, cb) {
 	    image_name, version_substr, service);
 
 	var onSearchFinish = function (err, image) {
-		if (err)
+		if (err) {
+			log.error(err);
 			return (cb(err));
+		}
 
 		log.info({ image: image }, 'found image %s for %s',
 		    image_name, service);
